mlir/lib/Dialect/LLVMIR/IR/LLVMDialect.cpp

```diff
@@ -146,6 +146,9 @@ static void printLLVMOpAttrs(OpAsmPrinter &printer, Operation *op,
   } else if (auto iface = dyn_cast<ExactFlagInterface>(op)) {
     printer.printOptionalAttrDict(filteredAttrs,
                                   /*elidedAttrs=*/{iface.getIsExactName()});
+  } else if (auto iface = dyn_cast<NonNegFlagInterface>(op)) {
+    printer.printOptionalAttrDict(filteredAttrs,
```

Comment:
I suspect this is not needed since the operation does not use `custom<LLVMOpAttrs>(attr-dict)`. It seems like this print/parseLLVMOpAttrs should only be used for the operations that use fast math flags. That can be cleaned up in a separate PR though. 

I brief I would suggest to remove the NonNegFlagInterface here, except it is necessary for a reason I do not see right now?

Reply:
Done!